hakyll.cabal

```diff
@@ -217,6 +218,10 @@ Library
     wai-app-static       >= 3.1      && < 3.2,
     yaml                 >= 0.8.11   && < 0.12
 
+  If os(windows)
```

Comment:
Needed because `System.Directory.Internal` does not export `createHardLink` for Windows